_posts/2019-07-01-Subnero-at-Oceans19.md

```diff
@@ -17,7 +19,7 @@ The [Marine Technology Society](https://www.mtsociety.org/) and the [IEEE Ocea
 
 Subnero attends this event once a year, every year. Once again we made new acquaintances and caught up on the latest news from the industry. We were privileged to attend Dr John Potter’s tutorial on Janus and to do a demonstration using UnetStack3. Our booth was very well attended during the entire exhibition.
```

Comment:
We were privileged to help with Dr John Potter’s tutorial on JANUS and to do a demonstration of JANUS using UnetStack3. We had lots of activity at our booth during the entire exhibition as well.

Reply:
updated the content